components/style/style_adjuster.rs

```diff
@@ -554,6 +564,7 @@ impl<'a, 'b: 'a> StyleAdjuster<'a, 'b> {
         &mut self,
         layout_parent_style: &ComputedValues,
         flags: CascadeFlags,
+        cacheable: &mut bool,
```

Comment:
I don't love the outparam... Can't this just be the return value? Not a big deal though, I guess.

Reply:
I would think outparam makes things clearer here and at the callsite.